connector-email/src/main/java/com/redhat/cloud/notifications/connector/email/config/EmailConnectorConfig.java

```diff
@@ -93,6 +94,9 @@ public class EmailConnectorConfig extends ConnectorConfig {
     @ConfigProperty(name = RBAC_URL)
     String rbacURL;
 
+    @ConfigProperty(name = USER_PROVIDER_CACHE_EXPIRE_AFTER_WRITE, defaultValue = "600")
```

Comment:
We're currently caching users for 10 minutes in prod. Let's do the same in the connector.